CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - Add: check response obj before use it handling http commands
 - Upgrade NodeJS version from 8.16.1 to 10.17.0 in Dockerfile due to Node 8 End-of-Life 
+- Add: /api-docs endpoint providing swagger-based documentation of the HTTP protocol
```

Comment:
After clarifying that the swagger documentation refers only to southbound (https://github.com/telefonicaid/iotagent-ul/pull/412/files#r368682255), this line should be slightly modified:

```suggestion
- Add: /api-docs endpoint providing swagger-based documentation of the HTTP southbound interface
```

Reply:
fixed in 2c34c9ab082a7e0b43255d21f690ffed0157703d